_posts/2022-06-23-sampling.md

```diff
@@ -32,7 +32,7 @@ The next section focuses on introducing and implementing the four random samplin
 This is the most basic form of sampling and is performed as shown above in the previous section. Simple sampling means choosing randomly, _without replacment_, from your population, until you reach the desired number of samples.
 
 ### Systematic Sampling Method
-Systematic sampling isn't really "random", but rather chooses the nth observation, where n is the $\frac{total number of observations}{number of samples you'd like}$. So you may take the 5th, 10th, 15th and so on up to 100 for a total of 20 samples. This method has the drawback that if the data is organized in any meaningful way by a certain attribute, then you will introduce bias into your sample.
+Systematic sampling isn't really "random", but rather chooses the nth observation, where n is the \$\frac{total number of observations}{number of samples you'd like}\$. So you may take the 5th, 10th, 15th and so on up to 100 for a total of 20 samples. This method has the drawback that if the data is organized in any meaningful way by a certain attribute, then you will introduce bias into your sample.
```

Comment:
Systematic sampling isn't really "random", but rather chooses the nth observation, where n is the \$\frac{total number of observations}{number of samples you'd like}\$. So you may take the 5th, 10th, 15th and so on up to 100 for a total of 20 samples. This method has the drawback that if the data is organized in a meaningful way by a certain attribute, then you can introduce bias into your sample.